omniNotes/src/androidTest/java/it/feio/android/omninotes/tests/HamburgerTest.java

```diff
@@ -19,37 +24,56 @@ public void HamburgerViewTest() {
         AddTextNote.typePayTheBills();
 
         // Step: Click return button
+        AddTextNote.clickBackButton();
 
         // Step: Delete (swipe) the note with title "Pay the bills"
+        Home.deletePayTheBillsNote();
 
         // Step: Click on hamburger
+        AddTextNote.clickBackButton();
 
         // Verify: The button with name Archive is displayed and has sibling the button with name Notes
+        assertTrue("The button with name Archive is not displayed", Home.isArchiveButtonDisplayed());
+        assertTrue("The button with name Archive doesn't have sibling the button with name Notes", Home.isArchiveButtonSiblingWithNotesButton());
 
         // Step: Click on Archive button
+        Home.clickArchiveButton();
 
-        // Verify: Note with title "Pay the bills" is displayed"
+        // Verify: Note with title "Pay the bills" is displayed
+        assertTrue("Note with title \"Pay the bills\" is not displayed", Home.isReminderSetDisplayed());
 
         // Step: Delete (swipe) the note with title "Pay the bills"
+        Home.deletePayTheBillsNote();
 
         // Step: Click on hamburger
+        Thread.sleep(3000);
+        AddTextNote.clickBackButton();
 
-        // Verify: The button with name Trash is displayed below the button with name Notes
+        // Verify: The button with name Trash is displayed //???below the button with name Notes
+        assertTrue("The button with name Trash is not displayed", Home.isTrashButtonDisplayed());
 
         // Verify: The button with name Archive is NOT displayed
+        assertFalse("The button with name Archive is displayed", Home.isArchiveButtonDisplayed());
 
         // Step: Click on Trash button
+        Home.clickTrashButton();
 
         // Verify: Note with title "Pay the bills" is displayed"
+        assertTrue("Note with title \"Pay the bills\" is not displayed", Home.isReminderSetDisplayed());
 
         // Verify: List View has one child
+        assertEquals("List View doesn't have one child", 1, Home.getListViewNoOfChildren());
```

Comment:
You can use directly "EspressoMethos.getListViewNoOfChildren( your matcher ));".